test/e2e_node/oomkiller_linux_test.go

```diff
@@ -37,6 +38,26 @@ type testCase struct {
 	oomTargetContainerName string
 }
 
+const KubeReservedMemory = "1500Mi"
+
```

Comment:
I would add something in the name to make it clear it's different then the other OOMKiller test. Maybe name it OOMKiller for pod exceeding node allocatable? or something like that?